components/admin_real/desktop/Body.jsx

```diff
@@ -22,6 +23,7 @@ export default function Body({ view }) {
       return (
         <BodyFrame>
           <ReservationTableContextProvider>
+            <CreateReservationButton />
```

Comment:
pulled it out bc it does not belong on the other component (plus that ones has already styles applied to it)